### PR TITLE
first commit

### PR DESCRIPTION
**Pull request by jedgarpark**
first commit of pixel art slideshow code for Matrix Portal, example .bmp files

### CODE REVIEW

File: Pixel_Art_Slideshow/pixel_art_slideshow.py [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+    dwell=IMAGE_DURATION,
+    h_align=HorizontalAlignment.RIGHT,
+)
+last_advance = time.monotonic()
+
+
+def advance():
+    # pylint: disable=global-statement
+    global align_right, last_advance
+    align_right = not align_right
```
Comment (makermelissa) [inline Pixel_Art_Slideshow/pixel_art_slideshow.py:48]: I'm not sure if you want all images to alternate or only 32. If you only want 32 pixel width images to change this, you could load the image first, get the image with slideshow. current_image_name(), load it up, check the width, and then flip align_right only if it's less than 64. Kinda cumbersome, but slideshow doesn't seem to expose much. Or if you're happy with how it works now, then I'm good with it too.

Comment (makermelissa) [reply Pixel_Art_Slideshow/pixel_art_slideshow.py:48]: When I wrote this, I didn't realize it was going to be a mix of 32 and 64 pixel wide images.

Comment (jedgarpark) [reply Pixel_Art_Slideshow/pixel_art_slideshow.py:48]: happy with how it works now, thanks.

**Review by makermelissa — APPROVED** [review]
Looks good to me.

File: Pixel_Art_Slideshow/pixel_art_slideshow.py (hunk 1 above)
Comment (makermelissa) [reply Pixel_Art_Slideshow/pixel_art_slideshow.py:48]: Thanks

**Pull request merged**